lua/nvim-treesitter/parsers.lua

```diff
@@ -69,7 +69,7 @@ end
 ---@field experimental boolean|nil
 ---@field readme_name string|nil
 
----@type ParserInfo[]
```

Comment:
shouldn't this still be type?

Reply:
oops, that's extraneous but I did that to shut lua_ls up :sweat_smile: